consensus/istanbul/backend/api.go

```diff
@@ -161,9 +162,19 @@ func (api *APIExtension) GetCouncil(number *rpc.BlockNumber) ([]common.Address,
 	}
 	// Ensure we have an actually valid block and return the council from its snapshot
 	if header == nil {
-		logger.Trace("Failed to find the requested block", "number", number)
 		return nil, errNoBlockExist // return nil if block is not found.
 	}
+
+	// Since v1.6.1, extra.validators represents a list of council
+	// TODO-Klaytn : replace this to the below calculation logic
```

Comment:
How is the blocks generated before v1.6.1 handled? As far as I understand, those blocks contain invalid committee list.

Reply:
As you understood, present blocks` validator list is wrong. 
So this code can return wrong Council member for the past block which is generated by present code.